rust/src/cliwrap/kernel_install.rs

```diff
@@ -64,6 +64,15 @@ fn redo_systemctl_wrap() -> Result<()> {
     Ok(())
 }
 
+#[context("Running initoverlayfs")]
+fn run_initoverlayfs() -> Result<()> {
+    const INITOVERLAY_INSTALL_CMD: &str = "initoverlayfs-install";
+
+    let empty_argv: Vec<&str> = Vec::new();
+    let _ = cliutil::exec_real_binary(INITOVERLAY_INSTALL_CMD, &empty_argv);
```

Comment:
We definitely don't want `cliutil::exec_real_binary` as that (as the name implies) calls the `exec` system call which will replace the current process.  That's not going to do what we want here.

How about just:

```suggestion
    let initoverlayfs_install = Path::new("/usr/bin/initoverlayfs-install");
    if initoverlayfs_install.try_exists()? {
        let st = Command::new(initoverlays_install).status()?;
        if !st.success() {
            anyhow::bail("Failed to invoke {initovelrayfs_install}");
        }
   }
```
or so

Reply:
Also the function could really use a comment saying what it does (at least a link to the upstream project), etc.